docs/GenericsManifesto.md

```diff
@@ -275,7 +275,7 @@ See the "Existentials" section, particularly "Generalized existentials", for mor
 
 ### Generalized supertype constraints
 
-Currently, supertype constraints may only be specified using a concrete class or protocol type.  This prevents us from abstracting over the supertype.
+Currently, supertype constraints may only be specified using a concrete class. This prevents us from abstracting over the value supertype, including existential types.
```

Comment:
Let's be careful about our terminology here.  You're attempting to clarify the wording around generalized existential bounds, but you've muddied the wording here and below around concrete protocol bounds in general.

Reply:
Not sure, if I understood you correctly, but let me try to answer. I've removed `or protocol type`, because currently `T: P` means protocol conformance, which is different from subtyping.

If `T` conforms to `P`, then `T` is a subtype of `any P`. `any P` is always a subtype of itself, but (in general case) `any P` does not conform to `P`.

Do you I understand correctly that by `bounds` you mean RHS side of both subtyping and conformance relation? The point I'm trying to make is, that the two are different things, and we should not generalise them.